lib/MySQL_Variables.cpp

```diff
@@ -0,0 +1,354 @@
+#include "MySQL_Variables.h"
+#include "proxysql.h"
+
+#include "MySQL_Session.h"
+#include "MySQL_Data_Stream.h"
+#include "SpookyV2.h"
+
+#include <sstream>
+
+MySQL_Variables::MySQL_Variables(MySQL_Session* _session) : session(_session), is_connected_to_backend(false) {
```

Comment:
`tx_isolation` is same as `isolation level` and tracked with same variable `isolation_level`
all other mysql variables may have several variables in `mysql_connection options` for value, for sent flag. These values are not necessary any more in current implementation. I checked variable correspondence and did not find any differences. Further we have automation test `./test/tap/tests/set_testing-p` that contains tests for all previously defined variables plus new character set variables `character_set_client`, `character_set_connection`, `character_set_database`

Reply:
`this` pointer can be null like in this example
```
#include <cstdio>

class A {
    public:
        int b;
        int a;
        void foo() {
            fprintf(stderr, "this %p\n", this);
        }
};

int main(void) {

    A *a=0;
    a->foo();
}
```
And it works

I did not see much projects that use `this` to point to each member variables. And also it takes extra 6 characters to type.

Reply:
@val214 I see nothing wrong in that example, `this` pointer is `null`, because the object isn't initialized, so the pointer table itself is null. If you try to refer to any internal member, like in, `fprintf(stderr, "this %p\n", **a**);` a segfault will occur, it doesn't matter if _this_ is being used (`this->a`), or it isn't (`a`). The explicit use of `this` is just an style choice.

I suggested it because it's simpler that establishing a convention, but if you feel like it's too verbose, we can talk in establishing a name convention for member variables, or do nothing if you feel like it's not worth the effort. Thanks you!